src/services/JwtTokenService.ts

```diff
@@ -0,0 +1,69 @@
+import jwt, { SignOptions } from "jsonwebtoken";
+
+interface JwtPayload {
+  userId: string;
+}
+
+interface TokenPair {
+  accessToken: string;
+  refreshToken: string;
+}
```

Comment:
Probably should be types? No inheritance needed, no class or API requirements. Just pure types.
Thus, could be put to /src/types.